engine/api/workflow/dao_node.go

```diff
@@ -273,6 +280,12 @@ func loadNodeContext(db gorp.SqlExecutor, store cache.Store, wn *sdk.WorkflowNod
 		}
 		ctx.Environment = env
 	}
+
+	if sqlContext.Conditions.Valid {
```

Comment:
you can use `gorpmapping.JSONNullString(sqlContext.Conditions, &ctx.Conditions)`